wordpress-mobile/WordPress-iOS
# Replacing Gridicon assets with Gridicons-iOS

This PR removes all image assets with 'gridicon' in the name and uses Gridicons-iOS to generate them at runtime.

To test:

* Check that the replaced images appear okay (most are in the blog detail menu). They should all be tinted, so should have the correct appearance.
* Definitely check an instance of `WPWebViewController`, as those buttons are created in Interface Builder but the images are set at runtime.
* Run the `review-deleted-images-for-commit` script to check the images aren't used anywhere.

@mattmiklic Would you mind checking that these look okay to you from a design point of view? The images are all displayed at default size, except for `gridicons-notice` was in the project at 20x20 (so I've stuck with that size for now). Not sure if you'll want to update this to 24?

Needs Review: @kurzee

Thank you!
